packages/opentelemetry-exporter-collector/README.md

```diff
@@ -0,0 +1,77 @@
+```
+
+## Usage in Web
+```js
+import * as opentelemetry from '@opentelemetry/core';
+import { SimpleSpanProcessor } from '@opentelemetry/tracing';
+import { WebTracer } from '@opentelemetry/web';
+import { CollectorExporter } from '@opentelemetry/exporter-collector'
+
+const collectorOptions = {
+  url: '<opentelemetry-collector-url>' // url is optional and can be omitted - default is http://localhost:55678/v1/trace
+};
+
+const tracer = new WebTracer();
+const exporter = new CollectorExporter(collectorOptions);
+tracer.addSpanProcessor(new SimpleSpanProcessor(exporter));
+
+opentelemetry.initGlobalTracer(tracer);
+
+```
+
+## Usage in Node
+```js
+const opentelemetry = require('@opentelemetry/core');
+const { BasicTracer, SimpleSpanProcessor } = require('@opentelemetry/tracing');
+const { CollectorExporter } =  require('@opentelemetry/exporter-collector');
+
+const collectorOptions = {
+  url: '<opentelemetry-collector-url>' // url is optional and can be omitted - default is http://localhost:55678/v1/trace
+};
+
+const tracer = new BasicTracer();
+const exporter = new CollectorExporter(collectorOptions);
+tracer.addSpanProcessor(new SimpleSpanProcessor(exporter));
+
+opentelemetry.initGlobalTracer(tracer);
+
+```
+
+## Running opentelemetry-collector locally to see the traces
+1. Go to examples/basic-tracer-node
+2. run `npm run collector:docker:ot`
+3. Open page at `http://localhost:9411/zipkin/` to observe the traces
+
+## Useful links
+- For more information on OpenTelemetry, visit: <https://opentelemetry.io/>
+- For more about OpenTelemetry JavaScript: <https://github.com/open-telemetry/opentelemetry-js>
+- For help or feedback on this project, join us on [gitter][gitter-url]
+
+## License
+
+Apache 2.0 - See [LICENSE][license-url] for more information.
+
+[gitter-image]: https://badges.gitter.im/open-telemetry/opentelemetry-js.svg
+[gitter-url]: https://gitter.im/open-telemetry/opentelemetry-node?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+[license-url]: https://github.com/open-telemetry/opentelemetry-js/blob/master/LICENSE
+[license-image]: https://img.shields.io/badge/license-Apache_2.0-green.svg?style=flat
+[dependencies-image]: https://david-dm.org/open-telemetry/opentelemetry-js/status.svg?path=packages/opentelemetry-exporter-collector
+[dependencies-url]: https://david-dm.org/open-telemetry/opentelemetry-js?path=packages%2Fopentelemetry-exporter-collector
+[devDependencies-image]: https://david-dm.org/open-telemetry/opentelemetry-js/dev-status.svg?path=packages/opentelemetry-exporter-collector
+[devDependencies-url]: https://david-dm.org/open-telemetry/opentelemetry-js?path=packages%2Fopentelemetry-exporter-collector&type=dev
+[npm-url]: https://www.npmjs.com/package/@opentelemetry/exporter-collector
+[npm-img]: https://badge.fury.io/js/%40opentelemetry%exporter-collector.svg
+[opentelemetry-collector-url]: https://github.com/open-telemetry/opentelemetry-exporter-collector
```

Comment:
```suggestion
[opentelemetry-collector-url]: https://github.com/open-telemetry/opentelemetry-collector
```

Reply:
We have all exporters starting in namespace `opentelemetry-exporter-*` originally it was meant to have name `opentelemetry-exporter-web-collector` but because `web` and `node` is one collector now I change this to be `opentelemetry-exporter-collector` and class `CollectorExporter`. Having just `opentelemetry-collector` would "destroy" the namespace . WDYT?

Reply:
I thought this url is to point Opentelemetry Collector repo. At least that's what it looks like "This module provides exporter for web and node to be used with opentelemetry-collector."

Reply:
ah yes you right, now I see it clearly :)